docs/articles/documentation/recipes/debugging-with-visual-studio-code.md

```diff
@@ -44,6 +45,7 @@ You will need to add the following configuration to the `launch.json` file.
 This configuration contains the following attributes:
 
 * `type` - specifies the type of the configuration. Set to `node` for a Node.js configuration.
+* `protocol` - specifies the Node.js [debugger wire protocol](https://code.visualstudio.com/docs/nodejs/nodejs-debugging#_supported-nodelike-runtimes). Set to `inspector` to use the [Inspector protocol](https://chromedevtools.github.io/debugger-protocol-viewer/v8/). This protocol is available in Node.js v6.3 (or v6.9 for Windows) or newer. Older Node.js versions do not support the `protocol` option. They use the `Legacy` protocol that has issues with source maps.
```

Comment:
`specifies the Node.js [debugger wire protocol](https://code.visualstudio.com/docs/nodejs/nodejs-debugging#_supported-nodelike-runtimes). Note that `inspector` protocol is supported in Node.js v6.3 (or v6.9 for Windows) or later. For early versions omit this property. In that case legacy debugger protocol will be used. Legacy protocol is well known for it's issues with source map support, therefore newer versions of Node.js are recommended.`

\cc @VasilyStrelyaev